docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
     depends_on:
       - db
   db:
-    image: postgres:10.3-alpine
+    image: postgres:9.4-alpine
```

Comment:
Was there a reason for the downgrade?

Reply:
When I ran `docker-compose up` the server wouldn't start, saying the migrations were generated for an earlier version of postgres. Maybe I misread the error and just needed to recreate the db container. I'll try again.

Reply:
Looks like we're using 10.6 in Heroku, so I'll figure this one out.